app/views/unattended/provisioning_templates/iPXE/preseed_default_ipxe.erb

```diff
@@ -36,7 +36,7 @@ ping --count 1 ${netX/dns} || echo Ping to DNS failed or ping command not availa
 <% kernel = boot_files_uris[0] -%>
 <% initrd = boot_files_uris[1] -%>
 
-kernel <%= kernel %> initrd=initrd.img interface=auto url=<%= url %> ramdisk_size=10800 root=/dev/rd/0 rw auto <%= snippet("preseed_kernel_options", variables: {ipxe_net: true}).strip %>
+kernel <%= kernel %> initrd=<%= initrd %> interface=auto url=<%= url %> ramdisk_size=10800 root=/dev/rd/0 rw auto <%= snippet("preseed_kernel_options", variables: {ipxe_net: true}).strip %>
```

Comment:
looking at the snapshot failures, this should be something like `basename(initrd)` (if such a thing exists)?

Reply:
or `initrd.split('/').last`

Reply:
The same functionality can be used here, too https://github.com/theforeman/foreman/blob/f0c5526b45cd2600281686e129382b3e1f2b72a8/app/views/unattended/provisioning_templates/iPXE/autoyast_default_ipxe.erb#L33